.github/workflows/continuous-integration-linux.yml

```
@@ -53,18 +53,26 @@ jobs:
             stdcxx: '20'
           - distro: 'ubuntu:latest'
             cxx: 'clang++'
-            cxx_extra_flags: '-fsanitize=address,undefined -fno-sanitize=function -fno-sanitize=vptr -fno-sanitize-recover=all'
-            extra_linker_flags: '-fsanitize=address,undefined -fno-sanitize=function -fno-sanitize=vptr -fno-sanitize-recover=all'
+            cxx_extra_flags: '-fsanitize=address -fno-sanitize=function -fno-sanitize=vptr -fno-sanitize-recover=all'
+            extra_linker_flags: '-fsanitize=address -fno-sanitize=function -fno-sanitize=vptr -fno-sanitize-recover=all'
             cmake_build_type: 'RelWithDebInfo'
             backend: 'THREADS'
             stdcxx: '23'
           - distro: 'ubuntu:latest'
             cxx: 'clang++'
-            cxx_extra_flags: '-fsanitize=address,undefined -fno-sanitize=function -fno-sanitize=vptr -fno-sanitize-recover=all'
-            extra_linker_flags: '-fsanitize=address,undefined -fno-sanitize=function -fno-sanitize=vptr -fno-sanitize-recover=all'
+            cxx_extra_flags: '-fsanitize=address -fno-sanitize=function -fno-sanitize=vptr -fno-sanitize-recover=all'
+            extra_linker_flags: '-fsanitize=address -fno-sanitize=function -fno-sanitize=vptr -fno-sanitize-recover=all'
             cmake_build_type: 'RelWithDebInfo'
             backend: 'SERIAL'
             stdcxx: '20'
+          - distro: 'ubuntu:latest'
+            cxx: 'clang++'
+            cxx_extra_flags: '-fsanitize=address -fno-sanitize=function -fno-sanitize=vptr -fno-sanitize-recover=all'
+            extra_linker_flags: '-fsanitize=address -fno-sanitize=function -fno-sanitize=vptr -fno-sanitize-recover=all'
```

```suggestion
            cxx_extra_flags: '-fsanitize=address'
            extra_linker_flags: '-fsanitize=address'
```
as far as I understand, these suppressions were for the undefined sanitizer

---

yeah just trying if it triggers